web/modules/custom/rdf_entity/src/Entity/Query/Sparql/Query.php

```diff
@@ -200,26 +199,29 @@ public function condition($property, $value = NULL, $operator = '=', $langcode =
         }
         return $this;
 
+      case '_field_exists-EXISTS':
+      case '_field_exists-NOT EXISTS':
+        $field_rdf_name = $this->getFieldRdfPropertyName($value, $field_storage_definitions);
+
+        if (!filter_var($field_rdf_name, FILTER_VALIDATE_URL) === FALSE) {
```

Comment:
It's better to use `\Drupal\Component\Utility\UrlHelper::isValid` to validate URLs.